modules/connected-to-disconnected-restore-insights.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/connected-to-disconnected.adoc
+
+[id="connected-to-disconnected-restore-insights_{context}"]
+= Restoring a degraded Insights Operator
+
+
+Disconnecting the cluster from the network necessarily causes the cluster to lose the internet connection. The Insights Operator becomes degraded because it requires access to link:https://console.redhat.com[Red Hat Insights].
+
+This topic describes how to recover from a degraded Insights Operator.
+
+.Procedure
+
+. Download the global cluster pull secret to your local file system:
++
+[source,terminal]
+----
+$ oc extract secret/pull-secret -n openshift-config --to=.
+----
```

Comment:
@umohnani8 The user doesn't need to do this stop, correct? They already have the `.dockerconfigjson` file and have edited it. Downloading the file again will overwrite their file. 

```
$ oc extract secret/pull-secret -n openshift-config --to=.
error: .dockerconfigjson: file exists, pass --confirm to overwrite
```

Reply:
Yeah, that looks correct, downloading the secret again would be overriding what they have done in the earlier steps. We can remove this step from the document.